delivery_carrier_label_default/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+{
+    "name": "Default label for carrier labels",
+    "summary": "This module defines a basic label to print "
+    "when no specific carrier is selected.",
+    "version": "14.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Delivery",
+    "website": "https://github.com/OCA/delivery-carrier",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": [
+        "base_delivery_carrier_label",
```

Comment:
Is this dependency really needed? Such a large one for a module that could stand alone...